src/main/kotlin/org/jitsi/videobridge/octo/OctoRelayServiceStats.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.videobridge.octo
+
+class OctoRelayServiceStats(
+    val bytesReceived: Long,
+    val bytesSent: Long,
+    val packetsReceived: Long,
+    val packetsSent: Long,
+    val packetsDropped: Long,
+    val receiveBitrate: Long,
+    val receivePacketRate: Long,
+    val sendBitrate: Long,
+    val sendPacketRate: Long,
+    val relayId: String
+) {
+
+    /**
+     * For java
+     */
+    data class Builder(
+        var bytesReceived: Long? = null,
+        var bytesSent: Long? = null,
+        var packetsReceived: Long? = null,
+        var packetsSent: Long? = null,
+        var packetsDropped: Long? = null,
+        var receiveBitrate: Long? = null,
+        var receivePacketRate: Long? = null,
+        var sendBitrate: Long? = null,
+        var sendPacketRate: Long? = null,
+        var relayId: String? = null
+    ) {
+        fun bytesReceived(bytesReceived: Long) = apply { this.bytesReceived = bytesReceived }
+        fun bytesSent(bytesSent: Long) = apply { this.bytesSent = bytesSent }
+        fun packetsReceived(packetsReceived: Long) = apply { this.packetsReceived = packetsReceived }
+        fun packetsSent(packetsSent: Long) = apply { this.packetsSent = packetsSent }
+        fun packetsDropped(packetsDropped: Long) = apply { this.packetsDropped = packetsDropped }
+        fun receiveBitrate(receiveBitrate: Long) = apply { this.receiveBitrate = receiveBitrate }
+        fun receivePacketRate(receivePacketRate: Long) = apply { this.receivePacketRate = receivePacketRate }
+        fun sendBitrate(sendBitrate: Long) = apply { this.sendBitrate = sendBitrate }
+        fun sendPacketRate(sendPacketRate: Long) = apply { this.sendPacketRate = sendPacketRate }
+        fun relayId(relayId: String) = apply { this.relayId = relayId }
+        fun build(): OctoRelayServiceStats = OctoRelayServiceStats(
```

Comment:
Adding any stats to this flow requires knowing to add it in multiple places already (`VideobridgeStatistics`).  I looked at doing a map-based approach as boris suggested, which would be nice, but it doesn't work because `VideobridgeStatistics` relies on filling in fields with default values if they're null.  

What did you have in mind?

Reply:
I'm not sure, it'd just be nice if missing a stat someplace was a compile-time error rather than a runtime error.

Reply:
I had that originally, but boris brought up the issue of it being easy to mismatch the arguments (since java doesn't have named arguments), so I went with what I thought was the closest Java equivalent which is a builder.

Reply:
I said above, but I thought about re-writing `OctoRelayService` in kotlin just to get the named args and then we can avoid this.  I could take a look at that?